vignettes/approx-inference.Rmd

```diff
@@ -171,7 +172,12 @@ fits <- list(
 
 draws <- imap(fits, function(fit, name) {
   predict_delay_parameters(fit) |>
-    draws_to_long() |>
+    as.data.frame() |>
```

Comment:
are you sure you need this as.data.frame call? It doesn't matter much as will go with a refactor to dplyr but might as well not include here if don't need.

Reply:
Yeah I think we need it for some reason (and that's bad)